pkg/state/conformance/state.go

```diff
@@ -847,5 +847,15 @@ func (suite *StateSuite) TestLabels() {
 	suite.Require().NoError(err)
 
 	suite.Require().Equal(1, list.Len())
-	suite.Assert().True(resource.Equal(path3, list.Get(0)))
```

Comment:
this is same issue - returned objects should contain correct version, no need to do any workarounds